packages/core/src/scss/components/foundation/forms/_forms.scss

```diff
@@ -216,12 +216,14 @@
 
     .loading-skeleton {
         animation: grows 150ms ease-out;
-        overflow: hidden;
         will-change: max-height;
     }
 
     @keyframes grows {
-        0% { max-height: $floating-label-input-height * 4; }
+        0% {
+            max-height: $floating-label-input-height * 4;
+            overflow: hidden;
+        }
         100% { max-height: 2000px; }
```

Comment:
I dont' see `overflow: visible; ` on 100%, is this working?

Reply:
fixed